web/channels/worker_channel.ex

```diff
@@ -13,7 +25,8 @@ defmodule MicrocrawlerWebapp.WorkerChannel do
 
   def handle_in("ping", payload, socket) do
     IO.puts "Received event - ping"
-    IO.puts Poison.encode_to_iodata!(payload, pretty: true)
+    IO.inspect self
+    # IO.puts Poison.encode_to_iodata!(payload, pretty: true)
```

Comment:
V pohode mohl jsi nechat zakomentovane, asi bysme mohli navazat na logger level debug nebo tak neco, at se to da snadno enablovat/disablovat.